src/java.desktop/share/classes/sun/print/PSPrinterJob.java

```diff
@@ -264,7 +264,7 @@ public class PSPrinterJob extends RasterPrinterJob {
    private AffineTransform mLastTransform;
 
    private double xres = PS_XRES;
```

Comment:
This change has absolutely nothing to with the bug, but I just happened to notice it as I was bumbling around the various sources.
Its a pure clean up since both are "300", and it just seemed to me it wasn't worth the overhead of a bug of its own so I include it here.